python/runtime/cudaq/spin/py_spin_op.cpp

```diff
@@ -103,9 +103,14 @@ void bindSpinOperator(py::module &mod) {
       .def("distribute_terms", &cudaq::spin_op::distribute_terms,
            "Return a list of `SpinOperator` representing a distribution of the "
            "terms in this `SpinOperator` into equally sized chunks.")
-      .def_static("random", &cudaq::spin_op::random,
-                  "Return a random spin_op on the given number of qubits and "
-                  "composed of the given number of terms.")
+      .def_static(
+          "random", &cudaq::spin_op::random,
+          "Return a random SpinOperator on the given number of qubits and "
+          "composed of the given number of terms.")
+      .def_static("from_word", &cudaq::spin_op::from_word,
+                  "Return a `SpinOperator` corresponding to the provided Pauli "
```

Comment:
I think it would be cleaner to provide the full Google doc style string here with args annoated. Then add the `XYY` example you mention as an example code block